src/com/github/adambots/steamworks2017/intake/Intake.java

```diff
@@ -0,0 +1,35 @@
+package com.github.adambots.steamworks2017.intake;
+
+import org.usfirst.frc.team245.robot.Actuators;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+public class Intake {
+	
+	//TODO: Set MAX_CURRENT to a number once we get an exact value
+	public static double MAX_CURRENT;
+	public static double SPEED = 0.8;
+	
+	/*
+	 * Runs intakeMotor
+	 * @Param intakeButon
+	 * */
+	public static void intake(boolean intakeButton){
+		if (intakeButton == true){
+			Actuators.getIntakeMotor().set(SPEED);
+		} else {
+			Actuators.getIntakeMotor().set(0);
```

Comment:
Again, no magic numbers. Even though it's just a zero add a constant that's like STOP_MOTOR_VALUE or something to the constants class